pkg/podexecutor/staticpod.go

```diff
@@ -64,7 +64,13 @@ func (s *StaticPod) APIServer(ctx context.Context, etcdReady <-chan struct{}, ar
 	if err := images.Pull(s.PullImages, "kube-apiserver", s.Images.KubeAPIServer); err != nil {
 		return nil, nil, err
 	}
-
+	args = append(args,
+		`--basic-auth-file=""`, // explicitly setting to empty value per CIS 1.5.
```

Comment:
this will break how rke2 works i think, if i understand correctly rke2 process uses this basic auth file generated by k3s

Reply:
@ibuildthecloud had mentioned that this is required for k3s but not for RKE2 and can be skipped here.